### PR TITLE
blui 4284 error state verify code & create account

### PR DESCRIPTION
**Pull request by JeffGreiner-eaton**
Fixes blui-4284.

#### Changes proposed in this Pull Request:

- add useEffect & call the validator on verify code and create account
- Note: there is failing test for Create Password Screen
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- update verify code screen component line 45 to code?.length > 1
- yarn start:example
- click create account & accept eula
- add invalid email & blur (error displays)
- nav back to eula & back to create account
- verify error displays
- add valid email & navigate to verify email screen
- enter one character & blur (error displays)
- nav back to create account & back to verify email (error displays)

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/src/new-architecture/screens/CreateAccountScreen/CreateAccountScreenBase.tsx [hunk 1]
```diff
@@ -53,6 +53,12 @@ export const CreateAccountScreenBase: React.FC<
         },
         [emailValidator]
     );
+    useEffect(() => {
+        if (emailInput.length > 0) {
+            setShouldValidateEmail(true);
+            handleEmailInputChange(emailInput);
+        }
+    }, [handleEmailInputChange, emailInput]);
```
[the remark this answers is not in the document]
Comment (JeffGreiner-eaton) [reply login-workflow/src/new-architecture/screens/CreateAccountScreen/CreateAccountScreenBase.tsx:61]: ok yes we talked about that. So do you suggest to also use eslint-disable here as well? lint didn't like the empty array

[the remark this answers is not in the document]
Comment (huayunh) [reply login-workflow/src/new-architecture/screens/CreateAccountScreen/CreateAccountScreenBase.tsx:61]: Correct. If you know what you are doing, go ahead and suppress ESLint. Maybe leave a comment somewhere to explain why.

[the remark this answers is not in the document]
Comment (joebochill) [reply login-workflow/src/new-architecture/screens/CreateAccountScreen/CreateAccountScreenBase.tsx:61]: Yeah, this is a very common case where we need to disable that lint rule (when we only want to run a useEffect once onMount).

**Comment by joebochill** [comment]
These two look good. We need to add the same thing for the validators on the firstName and lastName fields in AccountDetails.

**Comment by joebochill** [comment]
I'll merge it now, we can make a follow-up PR with AccountDetails addition

**Review by joebochill — APPROVED** [review]

**Pull request merged**